src/lib/services/licence-gauging-stations-service.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const licenceGaugingStationsRepo = require('../connectors/repos/licence-gauging-stations');
+
+const createNewLicenceLink = async (gaugingStationId, licenceId, properties = {
+  licenceVersionPurposeConditionId: null,
+  thresholdUnit: 'mAOD',
+  thresholdValue: 0,
+  abstractionPeriod: null,
```

Comment:
Does there need to be some code to map the `abstractionPeriod` object above, e.g.:
```
{
  startDay: 1,
  startMonth: 1,
  endDay: 31,
  endMonth: 12
}
```
To the DB fields `abstraction_period_start_day` etc...

Reply:
You're totally right. I will flatten this object and correct the naming mismatch. Thank you for spotting this.

Reply:
Resolved in 5c9be4f8